distributionviewer/core/static/js/app/actions/metric-actions.js

```diff
@@ -1,6 +1,5 @@
 import * as types from './action-types';
 
-
```

Comment:
Ok. I don't think I meant to change this b/c I left all the others. But I'll fix0r it.